src/styles/calendar.scss

```diff
@@ -0,0 +1,50 @@
+$size: 100px;
+$month-length: 31;
+$sun: 606px;
+
+.calendar {
+  gap: 1px;
+  display: flex;
+  box-sizing: border-box;
+  width: 726px;
+  height: fit-content;
+  flex-wrap: wrap;
+
+  position: absolute;
```

Comment:
It's not recommended to use the `position: absolute` property unless it's necessary. It removes an element from the normal document flow, and can cause issues with overlap or with the positioning of other elements.